database-service/tests/conftest.py

```diff
@@ -24,14 +27,16 @@ def app_root_folder(request):
 
 
 @pytest.fixture
-def hasura_client(hasura_endpoint, app_root_folder):
-    client = HasuraClient(hasura_endpoint)
-    yield client
-    client.rollback_migrations(app_root_folder)
+def fixtures_set(request, app_root_folder, hasura_client):
+    option = request.config.getoption("--fixtures-set")
+    fixtures_project_folder = os.path.join(
+        app_root_folder, 'fixtures', 'small')
+    yield fixtures_project_folder
+    hasura_client.rollback_migrations(fixtures_project_folder)
```

Comment:
`small_fixtures` is replaced with `fixtures_set`. It can be that we want to play with other fixtures sets in our tests, so that is more flexible here.